queue_stack.py

```diff
@@ -0,0 +1,48 @@
+"""
+Time Complexity: O(n)
```

Comment:
Please write time complexity of individual functions and try to explain your time complexity like what n stands for here